### DISCUSSION TITLE
Are video events inventoried?

### DISCUSSION THREAD

**Discussion by annelinja** [bdd100k/bdd100k#285] [Q&A] [answered]
Is there a list of the events that occur in the videos? For example, "following fire truck", "turning right", "residential area".

**Comment by fyu** [comment, marked as answer]
Nope. But events like "turning right" can be extracted from map overlay: https://doc.bdd100k.com/usage.html#trajectories

**Reply by annelinja** [reply]
Got it. Thanks for your help
